src/dev/performance/run_performance_cli.ts

```diff
@@ -42,7 +42,14 @@ interface JourneyTargetGroups {
 const journeyTargetGroups: JourneyTargetGroups = {
   kibanaStartAndLoad: ['login'],
   crud: ['tags_listing_page', 'dashboard_listing_page'],
-  dashboard: ['ecommerce_dashboard', 'data_stress_test_lens', 'flight_dashboard'],
+  dashboard: [
+    'ecommerce_dashboard',
+    'data_stress_test_lens',
+    'flight_dashboard',
+    'web_logs_dashboard',
+    'web_logs_dashboard_dataview',
+    'web_logs_dashboard_esql',
```

Comment:
We can't include all of dashboard journeys: it takes around 3 min to run each of them 2 times (warm & test), so having six is `18-20 min`

Having it in place, will limit how many PRs can trigger performance tests. Maybe we shouldn't target bare metal and test on regular ones? it probably will be less consistent and not comparable 1:1 with current main runs, but we can compare it with the same runs on regular workers done every commit. Thoughts?

Reply:
oki sg thx understood.

any thoughts on me swapping `ecommerce_dashboard` for `web_logs_dashboard`? This will align it a little more with the 3 main journeys we're keeping track of for dashboards (`web_logs_dashboard`, `data_stress_test_lens`, `flight_dashboard`)